mkdocs.yml

```diff
@@ -32,7 +32,7 @@ nav:
                 - 'Systems Administration': handbook/systems-administration.md
                 - 'Network Communications': handbook/network-communications.md
                 - 'Appendix': handbook/appendix.md
-                - 'Community Contributed Tutorials': handbook/community.md
+                - 'Community Contributed Tutorials (Oracle, Squeak, ...) ': handbook/community.md
```

Comment:
Leave the 'Community Contributed Tutorials' as it is, do not add Oracle and Squeak.